tests/mt_api/mt_api_test.py

```diff
@@ -810,3 +810,135 @@ def test_deepl_bulk_mt_no_target_language(
                 page_translation[target_language_slug] is None
                 or page_translation[target_language_slug].machine_translated is False
             )
+
+
+do_not_translate_title = [True, False]
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("do_not_translate_title", do_not_translate_title)
+def test_do_not_translate_title(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    do_not_translate_title: bool,
+    settings: SettingsWrapper,
+    caplog: LogCaptureFixture,
+    mock_server: MockServer,
+) -> None:
+    """
+    Check `do_not_translate_title` flag works as expected
+
+    :param load_test_data: The fixture providing the test data (see :meth:`~tests.conftest.load_test_data`)
+    :param login_role_user: The fixture providing the http client and the current role (see :meth:`~tests.conftest.login_role_user`)
+    :param do_not_translate_title: The value of `do_not_translate_title` flag
+    :param settings: The fixture providing the django settings
+    :param caplog: The :fixture:`caplog` fixture
+    :param mock_server: The fixture providing the mock http server used for faking the DeepL API server
+    """
+    client, role = login_role_user
+
+    if role in [*PRIV_STAFF_ROLES, MANAGEMENT, EDITOR]:
+        region = Region.objects.filter(slug=REGION_SLUG).first()
```

Comment:
Could you please explain why you use `filter` instead of `get()` or `get_object_or_404`?

Reply:
No deep meaning 😃

Reply:
Ok, in theory this could cause troubles. But I don't think it will in practice :thinking: